op-chain-ops/deployer/keygen.go

```diff
@@ -0,0 +1,35 @@
+package deployer
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum-optimism/optimism/op-chain-ops/devkeys"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type KeyGenerator interface {
+	Address(key devkeys.Key) (common.Address, error)
+	PrivateKey(key devkeys.Key) (*ecdsa.PrivateKey, error)
+}
+
+type MnemonicKeyGenerator struct {
+	dk *devkeys.MnemonicDevKeys
+}
```

Comment:
The `devkeys` package already provides interfaces, and the mnemonic keys approach that implement the interface. Do we really need this keygen wrapper?

Reply:
Removed.